kv.go

```diff
@@ -629,6 +630,14 @@ func (kv *kvs) Create(key string, value []byte) (revision uint64, err error) {
 		return kv.Update(key, value, e.Revision())
 	}
 
+	// Check if the expected last subject sequence is not zero which implies
+	// the key already exists.
+	if aerr, ok := err.(*APIError); ok {
+		if aerr.ErrorCode == 10071 {
```

Comment:
something that we started doing in the nats.go client is to move these codes into `jserrors.go`:
https://github.com/nats-io/nats.go/blob/main/jserrors.go#L121

then to make it compatible with `errors.Is` defining something like:

```
ErrWrongLastSequence JetStreamError = &jsError{apiErr: &APIError{ErrorCode: JSErrStreamWrongLastSequence}}
```
then this check would become:

```go
if errors.Is(err, ErrWrongLastSequence) {
 return ErrKeyExists
}
```

What I think might be helpful eventually is to have something like another `KeyValueError` type that embeds the original JS APIError to be able to inspect the original JS API Error which resulted into the KeyValueError by using `errors.As`.